docs/sources/auth/ldap.md

```diff
@@ -199,6 +201,13 @@ group_search_filter = "(member:1.2.840.113556.1.4.1941:=CN=%s,[user container/OU
 group_search_filter = "(|(member:1.2.840.113556.1.4.1941:=CN=%s,[user container/OU])(member:1.2.840.113556.1.4.1941:=CN=%s,[another user container/OU]))"
 group_search_filter_user_attribute = "cn"
 ```
+
+This config should work out of the box
```

Comment:
What should this configuration do? I suggest something like:

To define a new group search filter:

Avoid idioms like "out of the box" which might be difficult for non-native English speakers to understand.

Reply:
changed in lates commit